dali/base/dafdesc.cpp

```diff
@@ -2366,6 +2366,29 @@ IFileDescriptor *createFileDescriptor(const char *lname, const char *clusterType
     return fileDesc.getClear();
 }
 
+IFileDescriptor *createFileDescriptor(const char *lname, const char *planeName, unsigned numParts)
+{
+    Owned<IStoragePlane> plane = getStoragePlane(planeName, true);
+    if (!numParts)
+        numParts = plane->numDefaultSprayParts();
```

Comment:
can getStoragePlane() return nullptr and so should be checked ?

Reply:
no, because the 2nd param to getStoragePlane (=true) means throw an exception if missing.